syntaxes/yaml.tmLanguage.json

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
What purpose does this file serve?

Reply:
This file provides grammar which is used for syntax validation.

Reply:
what syntax validation? yaml or workflow?

Reply:
looks like yaml syntax. so, the server does not provide this syntax validation?

Reply:
Thie syntax validation is for YAML files, in general, and not specifically for workflow files. The server provides only those features which are schema dependent like auto-completion and hover support.